openedx/openedx-wordpress-ecommerce
# fix: file and class names corrected based on wordpress requirements

## Description

The file names were changed due to WordPress restrictions when uploading the plugin, which prohibited the use of reserved words such as 'plugin' or 'woocommerce.' It was decided that this name change would be made to 'openedx_ecommerce' instead of 'openedx_woocommerce_plugin.' Similarly, in accordance with WordPress code best practices, the names of the classes and namespaces used in the code were also changed.

## Testing instructions

Pull the branch locally and run the 'composer install' command to install all plugin dependencies and test. Additionally, these changes have already been uploaded to the staging environment where you can verify that everything is functioning correctly.

## Checklist for Merge

- [x] Updated documentation / NA
- [x] Rebased master/main / NA
- [ ] Squashed commits / NA
